migrations/2023-11-03-131721_create_external_auth/up.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE external_auth (
+    id serial PRIMARY KEY,
+    local_site_id int REFERENCES local_site ON UPDATE CASCADE ON DELETE CASCADE NOT NULL,
```

Comment:
This seems unnecessary, its not used in any queries except as filter, and nobody would have external auth for a different local site.